tools/python/boutiques/schema/descriptor.schema.json

```diff
@@ -49,29 +49,33 @@
             },{
                 "oneOf": [{
                     "properties": {
-                        "type": { "enum": ["docker"] },
+                        "type": { "enum": ["docker", "singularity"] },
```

Comment:
This breaks backward compatibility, not a problem but we should mention it in the release notes.

Reply:
Good point - I added a comment in https://github.com/boutiques/boutiques/issues/106